assets/templates/bed.html

```diff
@@ -1 +1 @@
-<li><span class='bed-count-figure'>{{available}}</span> <span class='bed-count-label'>{{type}} beds available</span><input class='button' type='submit' value='Book 1 bed'><a href='#'>Ring-fence beds</a></li>
+<li><span class="bed-count-figure">{{available}}</span> <span class="bed-count-label">{{type}} beds available</span></li>
```

Comment:
should probably format through something that'll figure out the plural so, `1 bed` vs `0 beds`, `2 beds`